Polyhedron/demo/Polyhedron/Plugins/PCA/Affine_transform_plugin.cpp

```diff
@@ -417,11 +417,11 @@ void Polyhedron_demo_affine_transform_plugin::grid()
       Scene_item::Bbox b = item->bbox();
 
 
-      double x_t(CGAL::sqrt(CGAL::squared_distance(Kernel::Point_3((b.min)(0), (b.min)(1), (b.min)(2)),
+      double x_t(1.001*CGAL::sqrt(CGAL::squared_distance(Kernel::Point_3((b.min)(0), (b.min)(1), (b.min)(2)),
```

Comment:
I don't see the need of using `sqrt`+ `squared_distance`. AFAIS, a simple subtraction will give you the result with more precision.